data/src/test/java/com/linkedin/data/schema/TestSchemaToPdlEncoder.java

```diff
@@ -50,4 +50,109 @@ public void testEncodeRecordWithEmptyDataMapInProperty() throws IOException
     assertEquals(source.getProperties(), encoded.getProperties());
     assertEquals(source, encoded);
   }
+
+  @Test
+  public void testEncodeSortsNestedPropertyMap() throws IOException
```

Comment:
Can you do one more test on nested properties larger than 1 nested layer?

Reply:
Good idea! Added testEncodeSortsMultiLevelNestedPropertyMap.